### DISCUSSION TITLE
AWS secrets manager plugin

### DISCUSSION THREAD

**Discussion by brackend** [grafana/grafana#53022] [General]
Hi,

I can see as of 9.1 we can use the plugin: https://grafana.com/docs/grafana/next/setup-grafana/configure-security/configure-database-encryption/store-secrets-using-aws-secrets-manager-plugin/?src=blog&camp=timeshift_19&plcmt=nav-products-cta2&cta=self-managed

The doc says that 9.1 does not include all secrets just now.  But that it will in the future.  So..
1. Is there somewhere it specifies exactly which secrets would be covered - I know user passwords are not.
2. Does it include oauth secrets - assuming not.  But would be great if it does
3. If not, do you know where I could find out what the rough timeline would be to implement "all secrets"?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
